server/main.js

```diff
@@ -69,6 +94,7 @@ app.route('/candCampAddress').get(function (req, res) {
 app.route('/candbio').get(function (req, res) {
   var candbio = req.query.candId;
   vs.collectCandidateDetails(candbio).then(function (results) {
+    console.log(results);
```

Comment:
Do we still need this console.log?
If so, let's add some context before the variable.